src/testdrive/src/main.rs

```diff
@@ -122,6 +122,13 @@ struct Args {
     #[clap(long)]
     temp_dir: Option<String>,
 
+    /// CLI arguments that are converted to testdrive variables.
+    ///
+    /// Passing: `--arg-var foo=bar` will create a variable named 'arg.foo' with the value 'bar'.
+    /// Can be specified multiple times to set multiple variables.
+    #[clap(long)]
+    arg_var: Vec<String>,
```

Comment:
Can we call this just `--var`? The argument-ness is implied by the fact that it is an `arg` IMO.

Reply:
yeah, I wanted to namespace the generated args as something. But `var.<varname>` looks weird, all variables are variables, hence the `arg.<name>` because it's a variable that comes from an argument. And I wanted the flag to sort of self-reference that: `--arg-var` is meant to look like `arg.<var>`.

Reply:
I think naming the variables `arg.blah` is great, but a CLI flag that has `--arg` in the name just seems redundant! It makes perfect sense to me to have the `--var foo=bar` flag set the `${arg.foo}` var IMO.